pkg/controller/route/route_controller.go

```diff
@@ -188,34 +202,45 @@ func (rc *RouteController) reconcile(nodes []*v1.Node, routes []*cloudprovider.R
 									UID:       types.UID(nodeName),
 									Namespace: "",
 								}, v1.EventTypeWarning, "FailedToCreateRoute", msg)
+							klog.V(4).Infof(msg)
```

Comment:
Some context would be helpful e.g. `Infof("FailedToCreateRoute: %s", msg)`

Reply:
The context is 5 lines above it, scoped to the if statement.